static/css/style.css

```diff
@@ -173,19 +172,19 @@ strong {
 
 /* member */
 
-.sleepy_cat { 
+.sleepy_cat {
 	font-size: 12px;
```

Comment:
I agree with @ercchy - please remove this bit, it's a big spaghetti :spaghetti:, and the PR isn't even about the members page anyway. 

By the way, if you were to do this, you can use the `about_list` class that wraps all members ...